HelpSource/Classes/Array.schelp

```diff
@@ -365,10 +365,11 @@ Routine {
 	a.fork; \doneAll.postln;
 }.play;
 )
+::
 
-poll(trig, label, trigid)
+method::poll(trig, label, trigid)
```

Comment:
no need to add the argument names explicitly, and better not. The help system does this for us.

Reply:
ok. done.